docs/devguide/translation.rst

```diff
@@ -0,0 +1,43 @@
+.. _dev-translate:
+
+============
+Translations
+============
+
+The Python community is a diverse and cosmopolitan place. Our documentation
+should be available for users in many different languages and for many different
+locales.
+
+This document explains how to go about translating this documentation.
+
+Read The Docs
+-------------
+
+Currently we host our documentation on http://readthedocs.org. We use Sphinx to
+make the documentation happen.
+
+The relevant documentation for localisation in ReadTheDocs is here:
+
+https://docs.readthedocs.io/en/latest/localization.html#project-with-multiple-translations
+
+Put simply, each new translation is a separate project, presumably a Sphinx
+based repository on GitHub, that contains the translated resources and is linked
+to ReadTheDocs in the usual way - making sure its locale is set  appropriately.
+
+It's possible for this "core" project to link to these other projects as
+"translations of" this one.
+
+Read the Docs automatically figures out the relationships and displays the
+appropriate links between all the related translation projects.
+
+I want to Translate!
+--------------------
+
+Great stuff! Thank you (in advance) for your support!
+
+* Make a new Sphinx project that mirrors the structure of the ``docs`` directory in this project. Call the project something obvious like: ``microbit-micropython-es`` (for the Spanish translation).
```

Comment:
I don't have a solution, but I could imagine a simple script that would pull the changes from the master, merge them, and then find the conflict markers and replace them with some Sphinx syntax to make them render nicely (we could add some CSS to make the old translated text appear with some background, and display the updated English text on hover, or something like that). Of course replacing the conflict markers in a way that doesn't mangle the restructured text syntax is a non-trivial task, but I think I could write something that at least handles most of the cases.

Then we can have that script run as a part of `config.py` or as a commit hook somewhere. (Or, run it manually and then correct the cases where it breaks the rendering, if I can't get it to work properly all the time.) The thing is, you don't need to know the language to which it is translated, to do that, so it can be done by a more technical person.